projects/igniteui-angular/src/lib/directives/radio/radio-group.directive.ts

```diff
@@ -24,7 +25,7 @@ export const RadioGroupAlignment = mkenum({
     horizontal: 'horizontal',
     vertical: 'vertical'
 });
-export type RadioGroupAlignment = (typeof RadioGroupAlignment)[keyof typeof RadioGroupAlignment];
+export type RadioGroupAlignment = typeof RadioGroupAlignment[keyof typeof RadioGroupAlignment];
```

Comment:
@skrustev , are you sure these radio group changes aren't coming from some issue with the target branch change?